### PR TITLE
add fire button test

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/392891325557410/1207203028528349/f
Tech Design URL:
CC:

**Description**:
Adds fire button tests, optimises run script.

**Steps to test this PR**:
1.  Check this passes
2. run setup_ui_test.sh
3. run run_ui_test.sh release/firebutton.yaml

### CODE REVIEW

File: DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -138,10 +138,10 @@
     {
       "identity" : "swift-argument-parser",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/apple/swift-argument-parser.git",
+      "location" : "https://github.com/apple/swift-argument-parser",
       "state" : {
-        "revision" : "0fbc8848e389af3bb55c182bc19ca9d5dc2f255b",
-        "version" : "1.4.0"
+        "revision" : "46989693916f56d1186bd59ac15124caef896560",
+        "version" : "1.3.1"
```
Comment (loremattei) [inline DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Is this intentional?

Comment (brindy) [reply DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Not sure what is going on there. I built from Xcode and that's how it resolved.

Comment (brindy) [reply DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: AppleToolbox is not using exact dependencies so my Xcode decided it didn't need to update.

**Review by bwaresiak — APPROVED** [review]
LGTM!

**Pull request merged**